.github/ISSUE_TEMPLATE/b-bug-report.yml

```
@@ -0,0 +1,87 @@
+name: Report a bug
+description: You encountered a bug in the geolocator plugin that you would like to report.
+title: "Bug: "
+labels: ["needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to submit an issue!
+        To effectively tackle it, we require as much information as possible.
+  - type: checkboxes
+    attributes:
+      options:
+      - label: I have searched the [existing issues](https://github.com/baseflow/flutter-geolocator/issues)
+        required: true
+  - type: checkboxes
+    attributes:
+      label: Please select affected platforms
+      options:
+      - label: Android
+      - label: iOS
```

This should list all supported platforms:
```suggestion
      - label: Android
      - label: iOS
      - label: Linux
      - label: macOS
      - label: Web
      - label: Windows
```